$MirOS: ports/print/cups/patches/patch-pdftops_Makefile,v 1.1 2006/06/21 19:25:35 bsiegert Exp $
--- pdftops/Makefile.orig	Wed Jul 11 23:46:19 2007
+++ pdftops/Makefile	Tue Oct 21 22:56:23 2008
@@ -123,7 +123,7 @@ uninstall:
 
 pdftops:	libxpdf.a pdftops.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CXX) $(LDFLAGS) -o $@ pdftops.o libxpdf.a $(LIBS) $(CXXLIBS) -lm
+	$(LIBTOOL) --mode=link $(CXX) $(LDFLAGS) -o $@ pdftops.o libxpdf.a $(LIBS) $(CXXLIBS) -lm
 
 
 #
